src/backend/opengl/font_impl.cpp

```diff
@@ -154,7 +156,7 @@ void font_impl::loadAtlasWithGlyphs(const size_t pFontSize)
         //    FT_Stroker_Done(stroker);
         //    FT_Done_Face(face);
         //    FT_Done_FreeType(library);
-        //    FT_THROW_ERROR("FT_Glyph_Stroke", fg::FG_ERR_FREETYPE_ERROR);
+        //    FT_THROW_ERROR("FT_Glyph_Stroke", forge::FG_ERR_FREETYPE_ERROR);
         //}
         //FT_Stroker_Done(stroker);
```

Comment:
It is related to outline strokes for characters, I added a `FIXME` label before the commented out code block. Removed it now.